src/Filter/PathsFilter.php

```diff
@@ -44,7 +44,10 @@ public function __construct(array $paths)
     public function isFeatureMatch(FeatureNode $feature)
     {
         foreach ($this->filterPaths as $path) {
-            if (str_starts_with(realpath($feature->getFile()), $path)) {
+            if (($filePath = $feature->getFile()) !== null
+                && ($realPath = realpath($filePath)) !== false
```

Comment:
(PHPStan) `realpath()` might fail and return false (which in turn doesn't work with `str_starts_with()`)